closed/openj9_version_info.h.in

```diff
@@ -30,5 +30,6 @@
 #define J9VERSION_STRING          "@VERSION_STRING@"
 #define OPENJDK_SHA               "@OPENJDK_SHA@"
 #define OPENJDK_TAG               "@OPENJDK_TAG@"
+#define JVM_VERSION_STRING        "openj9-@OPENJ9_BRANCH@-@OPENJ9_SHA@"
```

Comment:
The "openj9-" prefix isn't required. As per https://github.com/eclipse/openj9/blob/master/doc/processes/release_process.md, for a release the branch names are of the form `openj9-#releaseNumber#` so with the "openj9-" prefix the string will look like `openj9-openj9-#releaseNumber#`.

I'm not sure what OPENJ9_BRANCH is going to be (is it really the branch or would it be a tag? if so it should be reanamed). Better to use the tag, and only include the SHA if there is no tag. It may be confusing to show the branch name, as the branch for a release would be (for example) "openj9-0.8.0" but the tag would be "openj9-0.8.0RC1".

Reply:
`OPENJ9_BRANCH` in this PR means the branch not tag. 
How about `OPENJ9_TAG` and use `OPENJ9_BRANCH-OPENJ9_SHA` when there is no `OPENJ9_TAG`?

Reply:
Updated the PR, pls have another look.

Reply:
Seems reasonable.